multimodal-search-tirg/flow-query.yml

```diff
@@ -0,0 +1,29 @@
+!Flow
+with:
+  prefetch: 10
+  read_only: true  # better add this in the query time
+  rest_api: true
+  port_expose: $JINA_PORT
+pods:
+  image_read:
+    uses: pods/image-read.yml
+    read_only: true
+    parallel: $PARALLEL
+  normalize:
+    uses: pods/normalize.yml
+    read_only: true
+  text_read:
+    uses: pods/query-text-filter.yml #filterql text modality
+    read_only: true
+    needs: [gateway]
+  encode:
```

Comment:
here we need a `uses_before: _merge_all` otherwise we cannot collect the different modalities